db/rdbms-schema/src/main/java/io/camunda/db/rdbms/LiquibaseScriptGenerator.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.db.rdbms;
+
+import com.github.vertical_blank.sqlformatter.SqlFormatter;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import java.util.Set;
+import liquibase.Liquibase;
+import liquibase.database.DatabaseFactory;
+import liquibase.resource.ClassLoaderResourceAccessor;
+import liquibase.sqlgenerator.SqlGeneratorFactory;
+
+public class LiquibaseScriptGenerator {
+
+  public static void main(final String[] args) throws Exception {
+    final var targetDir = args[0] + "/liquibase";
+    final var databases = Set.of("h2", "mysql", "postgresql");
+
+    for (final var database : databases) {
+      generateLiquibaseScript(
+          database, "db/changelog/rdbms-support/changelog-master.xml", targetDir, "master.sql");
+
+      generateLiquibaseScript(
+          database,
+          "db/changelog/rdbms-support/8_8_0/8.8.0_master.xml",
```

Comment:
Hm, I think we should discuss the chronology of the files here. When we think about further releases ...what do you think about something like that:

db/changelog/(rdbms-support*)/
- changesets/8_8_0_changsets.xml
- changesets/8_8_1_changsets.xml

* do we really need our own namespace here? rdbms-support is very generic, and we only need liquibase because of the rbdms_support. If we really have other modules sometimes, we should at least be more specifiv like rdbms_exporter.

Reply:
Agreed. Renamed it to rdbms-exporter. I could think of an RDBMS persistent session store (and who knows what's to come).

Future generated SQL scripts would then have a name schema like "8.8.0_to_8.8.1.sql" or similar. The master would still contain all.